net/spdy/fuzzing/hpack_fuzz_util_test.cc

```diff
@@ -116,7 +116,11 @@ TEST(HpackFuzzUtilTest, PassValidInputThroughAllStages) {
 
 TEST(HpackFuzzUtilTest, ValidFuzzExamplesRegressionTest) {
   base::FilePath source_root;
+#if defined(STARBOARD)
+  base::PathService::Get(base::DIR_SOURCE_ROOT, &source_root);
```

Comment:
I think this was part of another change too, where I suggested we should probably just implement `DIR_SOURCE_ROOT` and point it to the matching folder.

Now this test simply continues with an empty path for `source_root`...

Reply:
I'll add a TODO here.